content/client-server-api/modules/push.md

```diff
@@ -195,6 +184,12 @@ they are represented as a dictionary with a key equal to their name and
 other keys as their parameters, e.g.
 `{ "set_tweak": "sound", "value": "default" }`.
 
+{{% boxes/note %}}
+Older versions of the Matrix specification included the `dont_notify` and
+`coalesce` actions. These should both be considered no-ops (ignored, not
+rejected) if received from a client.
+{{% /boxes/note %}}
```

Comment:
My fear is that someone will reject a push rule with actions of `["dont_notify"]`, which is equivalent to `[]`.

Reply:
I'm trying to evaluate this change for a system where 1.3 and 1.11 servers will co-exists and am wondering: In terms of RFC2119-language, would the "ignoring without rejecting" behavior be a MUST or a SHOULD?

Reply:
Probably a MUST, old clients might still use `dont_notify`?

Reply:
Yeah, I think that would make sense. Contrary to what I wrote in https://github.com/matrix-org/matrix-spec/issues/1782#issuecomment-2188074209 this morning this might actually be a case where keyword usage in an info box would be helpful.